### PR TITLE
[Examples] Add the named model feature into pytorch demo.

### PR DESCRIPTION
**Pull request by CaptainVincent**
~~This PR is pending for the wasinn plugin 0.13.4 release.~~ Ready

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: pytorch-mobilenet-image/README.md [hunk 1]
```diff
@@ -77,3 +81,25 @@ Executed graph inference
    4.) [950](10.4899)orange
    5.) [953](9.4834)pineapple, ananas
 ```
+
+- Case 2: Apply named model feature
+> requirement wasi-nn >= 0.5.0 and WasmEdge-plugin-wasi_nn-(*) >= 0.13.4
+
+
+```bash
+wasmedge --dir .:. --nn-preload demo:PyTorch:CPU:mobilenet.pt wasmedge-wasinn-example-mobilenet-image-named-model.wasm demo input.jpg
```
Comment (juntao) [inline pytorch-mobilenet-image/README.md:90]: Can you explain what the `demo:PyTorch:CPU:mobilenet.pt` name means?

Comment (hydai) [reply pytorch-mobilenet-image/README.md:90]: This is defined in the named model feature: `--nn-preload`
Ref: https://github.com/WebAssembly/wasi-nn/issues/36

Comment (juntao) [reply pytorch-mobilenet-image/README.md:90]: But the encoding and target are determined by the API calls in the code. What if the information in the name conflicts with the settings in the code? Why do we have to repeat ourselves here?

Comment (CaptainVincent) [reply pytorch-mobilenet-image/README.md:90]: In fact, `load_by_name` doesn't have parameters to specify the target and encoding of building the graph. Similarly, you can't assign a name to a model loaded by `load_by_files` ( or `bytes`) (loading a model from WebAssembly). These two mechanisms, IIUC from the API definition, there don't have interoperability. By the way, there's a scenario where `--nn-preload` multiple models with conflicting names can occur (our implementation the later one will overwrite the earlier one).

Comment (CaptainVincent) [reply pytorch-mobilenet-image/README.md:90]: In fact, `wasi_ephemeral_nn::load_by_name` doesn't have parameters to specify the target and encoding of building the graph. Similarly, you can't assign a name to a model loaded by `load_by_files` ( or `bytes`) (loading a model from WebAssembly). These two mechanisms, IIUC from the API definition, there don't have interoperability. By the way, there's a scenario where `--nn-preload` multiple models with conflicting names can occur (our implementation the later one will overwrite the earlier one).

**Review by hydai — APPROVED** [review]

**Pull request merged**